content/training/R_Basics/Objects.Rmd

```diff
@@ -36,7 +36,12 @@ reporting_week <- "2018-W10"   # creates the object reporting_week by assigning
 
 reporting_week                 # prints the current value of reporting_week
 ```
-*IMPORTANT: **An object's value can be over-written** at any time by running an assignment command to re-define its value. Thus, the **order of the commands run is very important**.*
+
+{{% notice tip %}}
```

Comment:
Suggest to use info since this is more of a warning than a tip and the box color should reflect that (https://learn.netlify.com/en/shortcodes/notice/)

```suggestion
{{% notice info %}}
```